python/sglang/srt/layers/linear.py

```diff
@@ -20,8 +20,10 @@
 from vllm.model_executor.layers.linear import LinearBase
```

Comment:
do we fully re-sync this file?

Reply:
No, I just updated the part which is not compatible. Do we need to fully sync it?